app/models/energy/hvac.py

```diff
@@ -0,0 +1,45 @@
+"""HVAC Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
```

Comment:
I feel like we aren't using most of these imported modules (I don't see datetime anywhere). Let's only import the modules that are needed for this file.